plugins/emoticons.md

```diff
@@ -9,20 +9,24 @@ controls: toolbar button
 
 This plugin adds a dialog to the editor lets users insert emojis into {{site.productname}}'s editable area. The dialog can be invoked via a toolbar button - `emoticons` - or a dedicated menu item added as `Insert > Emoticons`.
 
-> Note: it doesn't automatically convert text emoticons into graphical smilies.
+The emoticons plugin provides an autocompleter for adding emoticons without using the toolbar button or menu item. Adding a colon `:`, followed by at least two characters will open an inline dialog showing matching emoticons.
+
+> **Note**: The emoticons plugin does not automatically convert text emoticons into graphical emojis.
```

Comment:
There's some inconsistency here with the use of `emoticons` and `emojis`.

Reply:
Fixed by https://github.com/tinymce/tinymce-docs/pull/1494/commits/4a7917b87cbf3216748ab8f1deba41caa1274003